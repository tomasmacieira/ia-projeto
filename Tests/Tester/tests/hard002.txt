ROW	5	2	2	1	1	2	0	5	0	2
COLUMN	3	2	0	3	2	2	0	5	1	2
2
HINT	0	7	L
HINT	7	9	C
