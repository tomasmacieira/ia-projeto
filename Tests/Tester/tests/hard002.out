t..t...Lmr
b..m......
...b...t..
.......m..
.......m..
....c..b..
..........
lr..lr...C
..........
.c...c....
